code/lib/core-events/src/errors/server-errors.ts

```diff
@@ -120,25 +120,6 @@ export class CouldNotEvaluateFrameworkError extends StorybookError {
   }
 }
 
-export class ConflictingStaticDirConfigError extends StorybookError {
-  readonly category = Category.CORE_SERVER;
-
-  readonly code = 1;
-
-  public readonly documentation =
-    'https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook-configuration';
-
-  template() {
-    return dedent`
-      Storybook encountered a conflict when trying to serve statics. You have configured both:
-      * Storybook's option in the config file: 'staticDirs'
-      * Storybook's (deprecated) CLI flag: '--staticDir' or '-s'
-      
-      Please remove the CLI flag from your storybook script and use only the 'staticDirs' option instead.
-    `;
-  }
-}
```

Comment:
I wonder if we can ever remove these errors, due to the incrementing number.

Reply:
I thought about that too! We don't have documentation for this so it wouldn't impact the URLs, however it would impact the telemetry data for this error category, in case we end up introducing a CORE_SERVER_0001 error in the future. Let's brainstorm!

Reply:
reminder to bring this back